### DISCUSSION TITLE
How to add last-modified date in footer

### DISCUSSION THREAD

**Discussion by peterschindler** [quarto-dev/quarto-cli#5504] [Q&A] [answered]
Dear Quarto community,

I was wondering how to get the `last-modified` date information printed out in the footer. I know how to get it into the title and I also tried using `{{< "meta" "last-modified" >}}`  in the `_quarto.yml` but nothing seems to work.
I appreciate the help!

Best regards,
Peter

**Comment by mcanouil** [comment, marked as answer]
There are no quotes in shortcodes or at least they are not needed. They only make the shortcode longer to type and less readable.
"last-modified" is not a YAML key, it's a value for `date`, thus if you want to get the value you need to call meta shortcode on date.

Below is an example using `format: revealjs`, since you did not specify any.
Hope this help as the same principle should work for any YAML key and for any formats.

<details><summary>See Quarto document code.</summary>

````qmd
---
title: revealjs lists
date: last-modified
footer: "{{< meta date >}}"
format: revealjs
---

## Slide

This is some text.

1. first
2. second

This is more text.

- bullet
- bill
````

</details>

<img width="573" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/2164c1ec-6de8-4a94-b3ee-c5efa94898cd">

**Reply by peterschindler** [reply]
I see - that makes sense! Thanks a lot for your help!

**Reply by mcanouil** [reply]
You're welcome!
